cmd2.py

```diff
@@ -1127,9 +979,11 @@ def __init__(self, completekey='tab', stdin=None, stdout=None, persistent_histor
         # Call super class constructor.  Need to do it in this way for Python 2 and 3 compatibility
         cmd.Cmd.__init__(self, completekey=completekey, stdin=stdin, stdout=stdout)
 
-        # Commands to exclude from the help menu or history command
-        self.exclude_from_help = ['do_eof', 'do_eos', 'do__relative_load']
-        self.excludeFromHistory = '''history edit eof eos'''.split()
+        # Commands to exclude from the help menu and tab completion
+        self.hidden_commands = ['eof', 'eos', '_relative_load']
```

Comment:
I like how you just provide the command name now and don't need to include the **do_***